package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "eslint.github.io",
+  "version": "1.0.0",
+  "description": "This contains the code running on eslint.org.",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "build": "webpack --module-bind json js/app/parser/index.js js/app/parser/index.built.js",
```

Comment:
Ahh.. ok, got it. I think this should be moved into Espree repository at some point (similar to browserify task in ESLint core), but I think we can merge it like this for now